changedetectionio/store.py

```diff
@@ -82,9 +82,8 @@ def __init__(self, datastore_path="/datastore", include_default_watches=True, ve
             if include_default_watches:
                 print("Creating JSON store at", self.datastore_path)
 
-                self.add_watch(url='http://www.quotationspage.com/random.php', tag='test')
-                self.add_watch(url='https://news.ycombinator.com/', tag='Tech news')
-                self.add_watch(url='https://changedetection.io/CHANGELOG.txt', tag='changedetection.io')
+                for i in range(50):
+                    self.add_watch(url='https://changedetection.io/CHANGELOG.txt?x='+str(i), tag='test')
```

Comment:
we add 50 sites to check, but just 10 threads check them concurrently from a queue